src/screens/Signup/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import SignupForm from '../../components/SignupForm';
+
+function Signup() {
+  return (
+    <>
```

Comment:
unnecessary fragment also unnecessary component, why you not call `SignupForm` component directly on root component